jwala-webservices/src/main/java/com/cerner/jwala/ws/rest/v1/service/group/impl/GroupServiceRestImpl.java

```diff
@@ -695,6 +701,7 @@ public Response generateAndDeployGroupAppFile(final String groupName, final Stri
             metaData = resourceService.getMetaData(groupAppMetaData);
             if (metaData.getEntity().getDeployToJvms()) {
                 // deploy to all jvms in group
+                historyFacadeService.write(hostName, group, "Deploying " + fileName + " to all JVMS in the group ", EventType.USER_ACTION_INFO, aUser.getUser().getId());
```

Comment:
Change the message to say the resource is only being deployed to the JVMs on the specified host. The comment on line 703 is actually wrong, this clause will only deploy the resource to all the JVMs on a host - I would fix the comment on line 703 as well.